sdk-extensions/aws/src/main/java/io/opentelemetry/sdk/extension/aws/trace/SamplingRuleApplier.java

```diff
@@ -50,7 +57,21 @@
   private final Matcher serviceTypeMatcher;
   private final Matcher resourceArnMatcher;
 
-  SamplingRuleApplier(GetSamplingRulesResponse.SamplingRule rule) {
+  private final Statistics statistics;
+
+  SamplingRuleApplier(String clientId, GetSamplingRulesResponse.SamplingRule rule) {
+    this.clientId = clientId;
+    ruleName = rule.getRuleName();
+    if (rule.getReservoirSize() > 0) {
+      // Until calling GetSamplingTargets, the default is to borrow 1/s if reservoir size is
```

Comment:
Ah I see - I forgot the server also returns the default rule. I was thinking about the localized default sampling rule, which is the same as a the default rule returned by the server, but is used as the initial sampler in X-Ray SDKs. This probably isn't too important since we should very quickly get the default rule from server, but we should err on the side of caution and still make the initial 0.05 as well (say a customer misconfigures their collector so it can't get rules from server, and is expecting a default 5% rate but gets 100% instead).

Reply:
Thanks! I was trying to come up with where the default of 0.05 is surfaced to customers (if it's not in any doc than the number could be anything) but realized it's surfaced as the Default on the console so will definitely make sense to go with it.

Reply:
For reference, here it is in the docs (spelled out instead of numerical, of course 🙃)

https://docs.aws.amazon.com/xray/latest/devguide/xray-concepts.html#xray-concepts-sampling